class-vip-support-user.php

```diff
@@ -400,6 +413,62 @@ public function filter_wp_redirect( $location ) {
 		return $location;
 	}
 
+	/**
+	 * Filters a users email address, removing the username if the email
+	 * is a VIP Support email alias.
+	 *
+	 * This helps simplify support email aliases in the users list so
+	 * they're not as lengthy and easier to understand at a glance.
+	 *
+	 * @param $email string The email address of a user.
+	 *
+	 * @return string
+	 */
+	public function filter_vip_support_email_aliases( $email ) {
+		if ( is_admin() && $this->is_vip_support_email_alias( $email ) ) {
```

Comment:
Noting for myself: in a separate PR we should extend this to include the machine user email address as well.